### PR TITLE
add image-to-ascii example

### PR DESCRIPTION
**Pull request by sophearak**

### CODE REVIEW

File: go-image-to-ascii/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/sophearak/image-to-ascii/master/assets/image-2-ascii.png" height="500" />
+</p>
+
+Image to ASCII is powered by `@now/go`, and helps you generate an ASCII image from a picture.
+
+## Deploy your own
+
+You'll want to fork this repository and deploy your own image to primitive.
```
Comment (sarupbanskota) [inline go-image-to-ascii/README.md]: > your own image to primitive.

Image to ASCII.

File: go-image-to-ascii/index.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+				w.Header().Set("Content-Type", "application/json")
+				w.Header().Set("Content-Length", strconv.Itoa(len(string(jm))))
+
+				w.Write(jm)
+			}
+		} else {
+			ui.Index(w)
+		}
+	}
+}
```
Comment (sarupbanskota) [inline go-image-to-ascii/index.go]: I'm not sure what our convention is, but the tab/spacing here seems very huge.

Can we stick to 2 spaces?

Comment (sarupbanskota) [reply go-image-to-ascii/index.go]: cc @timothyis

Comment (sophearak) [reply go-image-to-ascii/index.go]: we cannot change that, go by default have `go lint` package.
image

also it's looking good in text editor.

cc @sarupbanskota

File: go-image-to-ascii/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/sophearak/image-to-ascii/master/assets/image-2-ascii.png" height="500" />
+</p>
+
+Image to ASCII is powered by `@now/go`, and helps you generate an ASCII image from a picture.
+
+## Deploy your own
+
+You'll want to fork this repository and deploy your own Image to ASCII.
```
Comment (unicodeveloper) [inline go-image-to-ascii/README.md]: Run the following command `now init go-image-to-ascii` to fetch the example to your local machine.

File: go-image-to-ascii/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+1. Click the fork button at the top right of GitHub
+2. Clone the repo to your local machine with `git clone URL_OF_FORKED_REPO_HERE`
```
Comment (unicodeveloper) [inline go-image-to-ascii/README.md]: I am not sure they need to follow these long processes to get to run it locally. You can replace these steps with:

> Run the following command `now init go-image-to-ascii` to fetch the example to your local machine.

File: go-image-to-ascii/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/sophearak/image-to-ascii/master/assets/image-2-ascii.png" height="500" />
+</p>
+
+Image to ASCII is powered by `@now/go`, and helps you generate an ASCII image from a picture.
```
Comment (unicodeveloper) [inline go-image-to-ascii/README.md]: Can we link to a live demo somewhere on top here?

File: go-image-to-ascii/README.md (hunk 4 above)
Comment (sophearak) [reply go-image-to-ascii/README.md]: Ahh, I see. sure I'll update.

File: go-image-to-ascii/README.md (hunk 5 above)
Comment (sophearak) [reply go-image-to-ascii/README.md]: Sure, I'll add.

**Pull request merged**